### DISCUSSION TITLE
suggestion: QCefView abstract layer that works in mac without cef needed.

### DISCUSSION THREAD

**Discussion by lencil** [CefView/QCefView#12] [Ideas]
for some reasons, mac app based on cef cannot be put into Apple Store.
so I think if QCefView has an abstract layer with same api in Qt, implented with wkwebview?
in windows, there's Edge WebView2, still can do the same layer to reduce released package size.
![QCefView](https://user-images.githubusercontent.com/9926721/125933100-ec2fd7cb-b02e-42dd-9e8b-09d00bb03a74.png)

**Comment by tishion** [comment]
This is really a hugue requirements, but i will consider it if i have enough engergy and time.
